packages/react/test/specs/behaviors/textAreaBehavior-test.tsx

```diff
@@ -0,0 +1,18 @@
+import { textAreaBehavior } from 'src/lib/accessibility'
+
+describe('textAreaBehavior.ts', () => {
+  test('aria-disabled is undefined by default', () => {
+    const behavior = textAreaBehavior({})
+    expect(behavior.attributes['root']['aria-disabled']).toEqual(undefined)
+  })
+
+  test('is enabled', () => {
+    const behavior = textAreaBehavior({ disabled: false })
+    expect(behavior.attributes['root']['aria-disabled']).toEqual(false)
+  })
+
+  test('is disabled', () => {
+    const behavior = textAreaBehavior({ disabled: true })
+    expect(behavior.attributes['root']['aria-disabled']).toEqual(true)
```

Comment:
I think that this file is redundant as should be tested by comments:
https://github.com/stardust-ui/react/blob/master/.github/test-a-feature.md#behavior-tets

https://github.com/stardust-ui/react/pull/1897/files#r322706395